### PR TITLE
Document funding in pubspec.yaml

### PR DESCRIPTION
**Pull request by jonasfj**
Fixes https://github.com/dart-lang/site-www/issues/4183

### CODE REVIEW

**Comment by mit-mit** [comment]
LGTM

**Comment by mit-mit** [comment]
@atsansone I'd like to link to this from the blog post

File: src/tools/pub/pubspec.md [hunk 1]
```diff
@@ -332,6 +336,22 @@ publish_to: none
 {% endprettify %}
 
 
+### Funding
+
+Package authors may use the `funding` property specify a list of URLs that
+provide information on how users can help fund the development of the package.
```
Comment (sfshaza2) [inline src/tools/pub/pubspec.md:342]:
```suggestion
Package authors can use the `funding` property to specify a list of URLs that
provide information on how users can help fund the development of the package.
For example:
```

File: src/tools/pub/pubspec.md [hunk 2]
```diff
@@ -332,6 +336,22 @@ publish_to: none
...
+
+{% prettify yaml tag=pre+code %}
```
Comment (sfshaza2) [inline src/tools/pub/pubspec.md:344]: Hmmm. Is dart.dev still using prettify?

File: src/tools/pub/pubspec.md [hunk 3]
```diff
@@ -332,6 +336,22 @@ publish_to: none
...
+funding:
+ - https://www.buymeacoffee.com/example_user
+ - https://www.patreon.com/some-account
+{% endprettify %}
+
+If published to [pub.dev]({{site.pub}}) the links will be displayed on the
+package page. This aims to help users that wish to help fund the development
+of their dependencies.
```
Comment (sfshaza2) [inline src/tools/pub/pubspec.md:352]:
```suggestion
If published to [pub.dev]({{site.pub}}) the links are displayed on the
package page. This aims to help users fund the development
of their dependencies.
```

**Review by sfshaza2 — APPROVED** [review]
lgtm, with a couple edits.

**Comment by sfshaza2** [comment]
Can we land this? Or do we need to wait a bit?

File: src/tools/pub/pubspec.md [hunk 4]
```diff
@@ -332,6 +336,23 @@ publish_to: none
...
+
+Package authors can use the `funding` property to specify a list of URLs that
+provide information on how users can help fund the development of the package.
+For example:
+
+{% prettify yaml tag=pre+code %}
+funding:
+ - https://www.buymeacoffee.com/example_user
+ - https://www.patreon.com/some-account
+{% endprettify %}
```
Comment (parlough) [inline src/tools/pub/pubspec.md:349]:
````suggestion
```yaml
funding:
 - https://www.buymeacoffee.com/example_user
 - https://www.patreon.com/some-account
```
````

**Review by parlough — APPROVED** [review]

File: src/tools/pub/pubspec.md (hunk 2 above)
Comment (parlough) [reply src/tools/pub/pubspec.md]: Only in cases where we need to highlight code, so it's fine to switch this case to a normal markdown code block.

**Comment by parlough** [comment]
The pub.dev site already supports this field(see https://github.com/dart-lang/site-www/issues/4183#issuecomment-1224809731), so I'm going to go ahead and publish this. 

Thanks @jonasfj!

**Pull request merged**